egs/aishell/asr/simple_v1/decode.py

```diff
@@ -137,8 +137,8 @@ def main():
     lang_dir = Path('data/lang_nosp')
     symbol_table = k2.SymbolTable.from_file(lang_dir / 'words.txt')
     phone_symbol_table = k2.SymbolTable.from_file(lang_dir / 'phones.txt')
-    ctc_topo = build_ctc_topo(list(phone_symbol_table._id2sym.keys()))
-    ctc_topo = k2.arc_sort(ctc_topo)
+    ctc_topo = k2.arc_sort(build_ctc_topo(list(
```

Comment:
Please update it like the one in librispeech, which excludes disambiguation symbols from 
`phone_symbol_table`. 

Sorry that aishell has not been kept in sync with librispeech. We have been doing
experiments with the librispeech recipe.

Reply:
will do that.

Reply:
Like the one in
https://github.com/k2-fsa/snowfall/blob/1567a6553af8519e52fa86a2c6fd5aa6bf0c43f9/egs/librispeech/asr/simple_v1/mmi_bigram_decode.py#L210